stratifi-app/src/components/FAQ/FAQ.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+function FAQ({ props }) {
+  return (
+    <div>
+      <button
+        className="accordion"
+        style="
```

Comment:
Good job bro! But the inline styling isn't consistent with our agreement to use Tailwind. I understand, it's my first time too. But try it, I promise you won't  regret it